diff --git a/src/com/android/googlesearch/SuggestionProvider.java b/src/com/android/googlesearch/SuggestionProvider.java
index f4cc78c..76cc081 100644
--- a/src/com/android/googlesearch/SuggestionProvider.java
+++ b/src/com/android/googlesearch/SuggestionProvider.java
@@ -58,7 +58,7 @@ public class SuggestionProvider extends ContentProvider {
     private static final int HTTP_TIMEOUT_MS = 1000;
 
     // TODO: this should be defined somewhere
-    private static final String HTTP_TIMEOUT = "http.connection-manager.timeout";
+    private static final String HTTP_TIMEOUT = "http.conn-manager.timeout";
 
     private static final String SUGGESTION_ICON =
             "android.resource://com.android.googlesearch/" + R.drawable.magnifying_glass;
